### ISSUE TITLE
Does `no-refer-to-non-existent-id` not work properly in PHP?

### ISSUE THREAD

**Issue by spicato-kanayama** [markuplint/markuplint#748]
- Markuplint version: 3.2.0
- Parser lang: PHP (php-parser)
- Node.js version: `18.13.0`
- OS: MacOS

## Describe the bug
I have no problem with HTML, but when I use anchor links (#[id]) in php and give the corresponding ID to a specific element, I get a "not found" error.
![image](https://user-images.githubusercontent.com/93689462/224233385-54e39332-fe98-4464-a625-e5c04ef48d1a.png)

## Code Example

```html
	<main class="cMain">
		<h1>テスト</h1>

		<a href="#hoge">
			テストアンカーリンク
		</a>

		<div id="hoge"></div>
	</main>
```

## Steps To Reproduce

This is occurring during constant monitoring using the VSCode extension.

## The current behavior

Use `.markuplintrc` to disable `no-refer-to-non-existent-id`.

## The expected behavior

PHP works the same way as HTML

よろしくお願いします。。。！

**Comment by YusukeHirao** [comment]
@spicato-kanayama 

Thank you for your report.
I will fix it sequentially.
